### ISSUE TITLE
Add option for this behaviour to only activate on Shift+Enter rather than always

### ISSUE THREAD

**Issue by MaddyGuthridge** [valeriangalliat/vscode-continue-inline-comments#2]

**Comment by life-droid** [comment]
I use RStudio as well as VSCode and in RStudio, Shift+Enter stops the continuation of comments to the new line. I would actually prefer this method. Sadly no `onShiftEnterRules`

**Comment by jwbth** [comment]
Yeah, the whole point is that you don't need to insert `// ` *always*, but you need to insert `// ` *often*. Without optionality, this extension doesn't make much sense, at least for me.
